core-rust/state-manager/src/store/rocks_db.rs

```diff
@@ -150,232 +147,148 @@ impl RocksDBStore {
         db_opts.create_if_missing(true);
         db_opts.create_missing_column_families(true);
 
-        let transactional_db_opts = TransactionDBOptions::default();
-
         let column_families: Vec<ColumnFamilyDescriptor> = ALL_COLUMN_FAMILIES
             .into_iter()
             .map(|cf| ColumnFamilyDescriptor::new(cf.to_string(), Options::default()))
             .collect();
 
-        let db = TransactionDB::open_cf_descriptors(
-            &db_opts,
-            &transactional_db_opts,
-            root.as_path(),
-            column_families,
-        )
-        .unwrap();
+        let db = DB::open_cf_descriptors(&db_opts, root.as_path(), column_families).unwrap();
```

Comment:
If we want to change back to a transaction DB, I assume it's something we can change at runtime with the same database files? Or are the files themselves different?

Reply:
Good question, I'll give it a try

Reply:
yup, seems to work fine